dali/python/nvidia/dali/ops.py

```diff
@@ -449,6 +451,11 @@ def __call__(self, *inputs, **kwargs):
         return outputs
 
 
+def current_dali_stream():
+    """Get DALI's current CUDA stream."""
+    return CUDAStream(nvidia.dali.libpython_function_plugin.current_dali_stream())
+
+
 class PythonFunctionBase(with_metaclass(_DaliOperatorMeta, object)):
```

Comment:
I would add a documentation stating how code executed inside the PythonFunction needs to be synchronized when running on the GPU.

Reply:
Extended the operator docs